units/14-javascript2_2/sessions/1-hash/README.md

```diff
@@ -40,33 +39,33 @@ Students will show progress toward reaching the objective based on their perform
 
 ### Opening
 
-Today we will learn about hashes. This is important because hashes are a tool that programmers use to store a large set of unordered data. It connects to what we've previously learned because we will be able to perform array like operations with hashes. 
+Today we will learn about objects. This is important because objects are a tool that programmers use to store a large set of unordered data. It connects to what we've previously learned because we will be able to perform array like operations with objects. 
 
-Lets say I want to specify the capital of each of the countries listed in the array you made for the Do Now. How can I do that? You can do that using a hash.
+Lets say I want to specify the capital of each of the countries listed in the array you made for the Do Now. How can I do that? You can do that using an object.
 
 **Note: in JavaScript some people will refer to this as a hash and others will call it an object.**
 
 ### Introduction of new material ("I do")
 
-####Create and initialize a hash
+####Create and initialize an object
 
 ```
-var animalSounds = {"cow": "Moo", "cat": "Meow", "dog": "Woof"};
+var animalSounds = {cow: "Moo", cat: "Meow", dog: "Woof"};
 ```
 
-Let's break this down right to left. Note the **open curly brace** (`{`) and **close curly brace** (`}`). These braces specify the beginning and end of the hash. This particular hash has three **key-value** pairs. Note that each key-value pair is separated by a comma. We then take this hash and store in a variable called `animalSounds`.
+Let's break this down right to left. Note the **open curly brace** (`{`) and **close curly brace** (`}`). These braces specify the beginning and end of the object. This particular object has three **properties**. Note that each property has a **name** and a **value**, which are separated by a colon; each name-value pair is separated by a comma. We then take this object and store in a variable called `animalSounds`.
 
-Let's talk about the key-value pairs more. In this hash, `cow`, `cat`, and `dog` represent keys. `"Moo"`, `"Meow"`, and `"Woof"` represent those keys' respective values. These values happen to be strings.
+Let's talk about the properties more. In this object, `cow`, `cat`, and `dog` represent property names. `"Moo"`, `"Meow"`, and `"Woof"` represent those properties' respective values. These values happen to be strings.
 
-####Access and print an element of a hash
+####Access and print an element of an object
 
 ```
 console.log(animalSounds["cat"]);
```

Comment:
Another problem here—most JavaScript programmers would write `animalSounds.cat` here, and using the bracket form is generally considered bad style. Of course, brackets work in all situations (including when the property name is not hard-coded), but again, students will overwhelmingly see dot-notation in external materials. Also, the students have seen plenty of dot notation already (e.g. `console.log`), so this is a missed opportunity to connect those concepts.